README.md

```diff
@@ -51,6 +51,86 @@ $ rails db:migrate
  $ rails s
  ```
 
+## MySQL permissions setup
+
+You may see an error about MySQL permissions. If you do, you need to set up your user's permissions within MySQL.
+
+You'll want to log in to MySQL as `root` so you can make these changes. With MySQL running, do:
+
+```bash
+$ sudo mysql
+```
+Enter your password (the one you normally use with `sudo`, not a MySQL related password). You'll be in the MySQL console.
+
+Change to the system database for MySQL.
+
+```sql
+mysql> use mysql;
+```
+
+List the MySQL users.
+
+```sql
+mysql> select user from user;
++---------------+
+| user          |
++---------------+
+| <your name>   |
+| mysql.session |
+| mysql.sys     |
+| root          |
++---------------+
+```
+
+`<your name>` should be there; this should be the same name as the `username` listed in `config/database.yml`.
+
+Look at the grants given to your user.
+
+```sql
+mysql> show grants for <your name>@localhost;
+```
+
+If you're having permission errors you probably do not have the needed grants.
+
+Grant yourself all privileges on all tables on `crowdrescue_dev`.
+
+```sql
+mysql> GRANT ALL PRIVILEGES ON `crowdrescue_dev`.* TO '<your name>'@'localhost' WITH GRANT OPTION;
+```
+
+You should give yourself privileges to the test database as well.
+
+```sql
+mysql> GRANT ALL PRIVILEGES ON `crowdrescue_test`.* TO '<your name>'@'localhost' WITH GRANT OPTION;
+```
+
```

Comment:
You may also want to give yourself permission on `crowdrescue_prod`, the production environment database.

Reply:
would that involve ssh'ing into the live DB?

Reply:
This would just be ```GRANT ALL PRIVILEGES ON `crowdrescue_prod`.* TO '<your name>'@'localhost' WITH GRANT OPTION;```

Reply:
sorry, I meant would you have to do that to actually use those permissions?

Reply:
I believe so. Occasionally it's useful to run it as if it's production locally.

Reply:
ah ok. I don't think the prod database was created for me:

mysql> show databases;
+--------------------+
| Database           |
+--------------------+
| information_schema |
| crowdrescue_dev    |
| crowdrescue_test   |
| mysql              |
| performance_schema |
| sys                |
+--------------------+
6 rows in set (0.00 sec)

(Also, it may be the case that I only got the test database because I ran tests, so perhaps I should put a note about that?)

Reply:
To generate the production database you'd need to run `RAILS_ENV=production rails db:setup`.

Reply:
ah ok. I probably did that at some point for the test db too and just forgot about it. :/

I'll make notes about all of these to clarify.

Reply:
Great, thanks!